shared/common-adapters/text.native.js

```diff
@@ -20,6 +22,18 @@ class Text extends Component<void, Props, void> {
     openURL(this.props.onClickURL)
   }
 
+  _urlCopy = () => {
```

Comment:
The open URL action (via `openURL(...)` is a in the Text component, so it seemed to make sense for copying the link to also be universal?

I started plumbing the action all the way through from the Message component but thought it might be overkill. I could also pass in an option on whether the long press option is enabled or not (and only enable for chat messages).

Is seems nice to have links be copyable everywhere tho.